apps/moodlenet/templates/services/backend/secret.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: moodlenet
+    service: backend
+  name: "{{ moodlenet_secret_name }}"
+  namespace: "{{ project_name }}"
+data:
+  DATABASE_USER: "{{ MOODLENET_VAULT.POSTGRESQL_USER | default('moodlenet_user') | b64encode }}"
```

Comment:
could be in the configmap :man_shrugging: ?

Reply:
Since the value is stored in an ansible vault, it should be generated in a secret, right ?
And all our apps are like that.

Reply:
ok